### DISCUSSION TITLE
how to use bark transformed the text I provided to Voice

### DISCUSSION THREAD

**Discussion by Felixqian4160** [oobabooga/text-generation-webui#1570] [Q&A]
how to use bark transformed the text I provided to Voice

**Comment by BarfingLemurs** [comment]
install the extension
https://github.com/oobabooga/text-generation-webui/discussions/1532

**Comment by Felixqian4160** [comment]
I know how to use in text-generation-webui when model relay me， but I want to how to give bark one paragraph text，and then transformed into voice
